src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -32,10 +38,12 @@
     2) model = torch.compile(model, backend="openvino")
 """
 
-
 @register_backend
 @fake_tensor_unsupported
 def openvino(subgraph, example_inputs):
+    if (os.getenv("PYTORCH_TRACING_MODE") is not None):
+        if (os.getenv("PYTORCH_TRACING_MODE") == "TORCHFX"):
+            return fx_openvino(subgraph, example_inputs)
```

Comment:
Not sure that controlling through env var is really required here. For user it is simpler to switch by providing backend name: `fx_openvino` or `ts_openvino` with having reasonable default as just `openvino` where we should choose ourself what's the best.